packages/core/web-dev-server.config.mjs

```diff
@@ -6,13 +6,16 @@
 
 import { importMapsPlugin } from '@web/dev-server-import-maps';
 import { hmrPlugin, presets } from '@open-wc/dev-server-hmr';
+
 /** Use Hot Module replacement by adding --hmr to the start command */
 const hmr = process.argv.includes('--hmr');
 
 export default /** @type {import('@web/dev-server').DevServerConfig} */ ({
-  nodeResolve: true,
   open: './',
   watch: !hmr,
+  nodeResolve: {
+    exportConditions: ['development'],
+  },
```

Comment:
The `exportConditions` allows libraries like lit to provide a dev build to catch more errors during development. When building these libraries will default to prod optimized builds.